### PR TITLE
Add pipeline docker

### LINKED ISSUE #70: Add dockerfile to realtime_trains for aws
### Task:

- [x] Add dockerfile
- [x] Configure main function in realtime_trains.py for use in AWS

### User Story:
As a developer, I want to make an image for a service so that it can be used with other developers and be portable and flexible.

### PR DESCRIPTION
**Pull request by Zhi-704**
Close #70 by adding a dockerfile to the directory `realtime_trains`. Used for building a container, that can then be deployed on AWS as an image. Fixed logs function so that they are shown on AWS.

### CODE REVIEW

File: realtime_trains/realtime_trains.py [hunk 1]
```diff
@@ -8,11 +8,30 @@
...
+    try:
+        load_dotenv()
+        print("Pipeline has started.")
+        data = get_api_data_of_all_stations()
+        print("Extract finished.")
+        modified_data = process_all_stations(data)
+        print("Transformation finished.")
+        import_to_database(modified_data)
+        print("Load finished.")
+        print("Pipeline has finished.")
```
Comment (A-Faris) [inline realtime_trains/realtime_trains.py:34]: please change print to logs

**Review by A-Faris — APPROVED** [review]
Looks good, just remove print statements

File: realtime_trains/realtime_trains.py (hunk 1 above)
Comment (Zhi-704) [reply realtime_trains/realtime_trains.py:34]: There for a failsafe

**Review by fm1psy — COMMENTED** [review]
This looks good, well done

**Pull request merged**